### ISSUE TITLE
Missing key 'diameter' in ops

### ISSUE THREAD

**Issue by alex-attinger** [MouseLand/suite2p#190]
Hi, I installed the latest version of suite2p (master branch from github). After registration, when trying to view the registration binaries, I get the following error: 

Traceback (most recent call last):
  File "C:\code\suite2p\suite2p\gui2p.py", line 1183, in reg_window
    RW = reggui.BinaryPlayer(self)
  File "C:\code\suite2p\suite2p\reggui.py", line 373, in __init__
    self.openFile(fileName, True)
  File "C:\code\suite2p\suite2p\reggui.py", line 623, in openFile
    if (type(ops["diameter"]) is not int) and (len(ops["diameter"]) > 1):
KeyError: 'diameter'

It looks like in the latest version, the 'diameter' key no longer exists in ops. A quick fix for me was:

adding ('diameter' in ops) to the if statement in line 623 of reggui.py.

Otherwise it seems to work fine for me.

**Comment by carsen-stringer** [comment]
oops good catch, I've fixed that now in github and made a new pip. since diameter is no longer used for cell detection I've removed it, and there's a new parameter ops['aspect'] that replaces the function that diameter served here -- to change the aspect ratio of the movies in the gui to the "true" aspect ratio specified by ops['aspect']. this is for recordings in which the um/pixels are different in X and Y. it's still backward compatible and works with diameter for old processed data.
